MikuMikuLibrary/Databases/StageDatabase.cs

```diff
@@ -75,90 +83,198 @@ public enum StageEffect
         public int Field18 { get; set; }
         public StageEffect[] StageEffects { get; set; }
         public uint Id { get; set; }
+        public uint unkHash { get; set; }
```

Comment:
Rename unkHash to UnknownHash

Reply:
Actually, might be better to make it UnknownId instead since the hashes are IDs in reality.

Reply:
Or is that maybe the ID of the stage? Try taking the murmur hash of the stage name and see if it matches.